pylint/checkers/refactoring/len_checker.py

```diff
@@ -102,3 +102,17 @@ def visit_unaryop(self, node):
             and _is_call_of_name(node.operand, "len")
         ):
             self.add_message("len-as-condition", node=node)
+
+    @staticmethod
+    def base_classes_of_node(instance: astroid.nodes.ClassDef) -> List[astroid.Name]:
```

Comment:
Could you add docstring to explain a bit the role of this method?

Reply:
Sure, let me know if the docstring is'nt clear enough :)

Reply:
This is more efficient, more elegant and it works 👌